src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -205,10 +205,16 @@ def download_logs(
         with SuppressAndLog(assisted_service_client.rest.ApiException, KeyboardInterrupt):
             install_config = Path(output_folder) / "cluster_files" / "install-config.yaml"
             client.download_and_save_file(cluster["id"], "install-config.yaml", str(install_config))
```

Comment:
Don't we want to change `install-config.yaml` to `install-config.json`

Reply:
Lets handle it in assisted no ?

Reply:
I think we should change both but we can approve this for now